src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -979,8 +979,8 @@ object FunctionRegistry {
 
   })
 
-  val fetStruct = TStruct("pValue" -> TFloat64(), "oddsRatio" -> TFloat64(),
-    "ci95Lower" -> TFloat64(), "ci95Upper" -> TFloat64())
+  val fetStruct = TStruct("p_value" -> TFloat64(), "odds_ratio" -> TFloat64(),
```

Comment:
While you're doing this, can you make sure all p-values are `p_value`, odds ratios are `odds_ratio`, etc.

Reply:
I think I did, no?